pkg/services/store/object/object.proto

```diff
@@ -0,0 +1,242 @@
+syntax = "proto3";
+package object;
+
+option go_package = "./;object";
+
+// Will be replaced with something from the SDK
+message UserInfo {
+  int64 id = 1; // internal grafana ID
+  string login = 2; // string ID? 
+}
+
+// Raw saved object metadata
+message RawObject {
+  // Unique ID
+  string UID = 1;
```

Comment:
wouldn't we want to replace `UID` and `kind` props with the `EntityReference`?

Reply:
🤔 Maybe... the intention with reference for summaries is more broad than just an identifier -- it is more like a set of constraints. (eg, must have a datasource of type influx, but I don't actually care what UID)

Likely a final version will need to hold whatever happens with the GRN idea/structure